### PR TITLE
Added GetCryptoSnapshot()

### PR DESCRIPTION
**Pull request by ccnlui**
## Context
Add method to query crypto snapshot endpoint.

## TODO
- ~Add unit test~

## Test result
```
css, err := marketdata.GetCryptoSnapshot("ETHUSD", "CBSE")

(dlv) p css 
*github.com/alpacahq/alpaca-trade-api-go/v2/marketdata.CryptoSnapshot {                                                                                                                                    
        LatestTrade: *github.com/alpacahq/alpaca-trade-api-go/v2/marketdata.CryptoTrade {                                                                                                                  
                Timestamp: (*time.Time)(0xc000382730),                                                                                                                                                     
                Price: 4393.18,                                                                                                                                                                            
                Size: 0.04299154,                                                                                                                                                                          
                Exchange: "CBSE",                                                                                                                                                                          
                ID: 191026243,                                                                                                                                                                             
                TakerSide: "S",},                                                                                                                                                                          
        LatestQuote: *github.com/alpacahq/alpaca-trade-api-go/v2/marketdata.CryptoQuote {                                                                                                                  
                Timestamp: (*time.Time)(0xc000382780),                                                                                                                                                     
                Exchange: "CBSE",                                                                                                                                                                          
                BidPrice: 4400.64,                                                                                                                                                                         
                BidSize: 5.67990727,                                                                                                                                                                       
                AskPrice: 4400.65,                                                                                                                                                                         
                AskSize: 0.72309233,},                                                                                                                                                                     
        MinuteBar: *github.com/alpacahq/alpaca-trade-api-go/v2/marketdata.CryptoBar {                                                                                                                      
                Timestamp: (*time.Time)(0xc000380420),                                                                                                                                                     
                Exchange: "CBSE",                                                                                                                                                                          
                Open: 4393.62,                                                                                                                                                                             
                High: 4396.45,                                                                                                                                                                             
                Low: 4390.81,                                                                                                                                                                              
                Close: 4393.18,                                                                                                                                                                            
                Volume: 132.02049802,                                                                                                                                                                      
                TradeCount: 278,                                                                                                                                                                           
                VWAP: 4393.9907155981,},                                                                                                                                                                   
        DailyBar: *github.com/alpacahq/alpaca-trade-api-go/v2/marketdata.CryptoBar {                                                                                                                       
                Timestamp: (*time.Time)(0xc000380480),                                                                                                                                                     
                Exchange: "CBSE",                                                                                                                                                                          
                Open: 4329.11,                                                                                                                                                                             
                High: 4455.62,                                                                                                                                                                             
                Low: 4231.55,                                                                                                                                                                              
                Close: 4393.18,                                                                                                                                                                            
                Volume: 95466.0903448,                                                                                                                                                                     
                TradeCount: 186155,                                                                                                                                                                        
                VWAP: 4367.7642299555,},                                                                                                                                                                   
        PrevDailyBar: *github.com/alpacahq/alpaca-trade-api-go/v2/marketdata.CryptoBar {                                                                                                                   
                Timestamp: (*time.Time)(0xc0003804e0),                                                                                                                                                     
                Exchange: "CBSE",                                                                                                                                                                          
                Open: 4350.15,                                                                                                                                                                             
                High: 4433.99,                                                                                                                                                                             
                Low: 4261.39,                                                                                                                                                                              
                Close: 4329.11,                                                                                                                                                                            
                Volume: 152391.30635034,                                                                                                                                                                   
                TradeCount: 326203,                                                                                                                                                                        
                VWAP: 4344.2956259855,},}       
```

### CODE REVIEW

File: marketdata/rest.go [hunk 1]
```diff
@@ -1209,6 +1210,31 @@ func (c *client) GetLatestCryptoXBBO(symbol string, exchanges []string) (*Crypto
 	return &latestXBBOResp.XBBO, nil
 }
 
+// GetCryptoSnapshot returns the snapshot for a given crypto symbol
```
Comment (gnvk) [inline marketdata/rest.go:1213]: [minor] If you already mention the symbol in the docstring, you should maybe also explain why the exchange is a required argument.

File: marketdata/rest.go [hunk 2]
```diff
@@ -1209,6 +1210,31 @@ func (c *client) GetLatestCryptoXBBO(symbol string, exchanges []string) (*Crypto
...
+	q.Set("exchange", exchange)
+	u.RawQuery = q.Encode()
+
+	resp, err := c.get(u)
+	if err != nil {
+		return nil, err
+	}
+
+	var snapshot CryptoSnapshot
+
```
Comment (gnvk) [inline marketdata/rest.go:1230]: [superminor] I wouldn't add an empty here.

Comment (ccnlui) [reply marketdata/rest.go:1230]: Removed. But honestly, I like these super minor comments so that we keep the code base consistent!

**Review by gnvk — APPROVED** [review]
LGTM. Thanks @ccnlui

**Pull request merged**